perfect_divide.py

```diff
@@ -0,0 +1,17 @@
+"""
+ generator that find a perfect number
```

Comment:
1. Please document inside the function.
2. What is a "perfect number"? explain and define it in your description.